.gitignore

```
@@ -116,3 +116,6 @@ network-api/networkapi/frontend/*
 
 # Cypress visual regression testing
 cypress/videos
+
+# Selenium drivers
+*driver.*
```

```suggestion
*driver*
```
That will ignore drivers on non windows system.